docs/core/logging.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Logging
```

Comment:
I just noticed small inconsistency within the doc and between python/java docs. In python we use `Logger`, `Tracer`. In Java it is `Logging`, `Tracing`. Down below in the text we switch back to `Logger`. So I would suggest to change it to `Logger` and `Tracer` to keep it consistent.

Reply:
I will check this, I guess in Java its better to keep calling them `Logging` and `Tracing` since thats what annotations are called. I will check for places in docs where its used otherwise